deker/uri.py

```diff
@@ -52,6 +57,22 @@ def raw_url(self) -> str:
         url += quote(str(self.path), safe=":/")
         return url
 
+    @classmethod
+    def __get_servers_and_netloc(cls, netloc: str, scheme: str) -> Tuple[str, Optional[List[str]]]:
+        """Parse list of servers.
+
+        :param netloc: Netloc object
+        :param scheme: http or https
+        """
+        if "," not in netloc or scheme not in ["http", "https"]:
+            return netloc, None
```

Comment:
Why scheme is returned as None if it doesn't match http(-s)?

Reply:
It's not scheme. It's servers that are returned as None. If scheme is not http(s) e.g file://, then we don't have support for cluster. At least yet.